doc/source/serve/tutorials/object-detection.md

```diff
@@ -0,0 +1,50 @@
+(serve-object-detection-tutorial)=
+
+# Serving Object Detection Model
+Build an object detection application with Ray Serve.
```

Comment:
```suggestion
This tutorial will show you how to run an object detection application with Ray Serve.
```

Reply:
```suggestion
Run an object detection application with Ray Serve in this tutorial.
```

Reply:
I'm suggesting an action-oriented structure to the sentence. If you feel strongly about a declarative sentence, then I suggest: `This tutorial shows how to run an object detection application with Ray Serve.`